src/components/Dashboard/ProgressBar/ProgressBar.js

```diff
@@ -1,15 +1,48 @@
-import React from "react";
+import React, { Component } from "react";
 import styles from "./ProgressBar.scss";
 
-const ProgressBar = () => {
-  return (
-    <ul className={styles.ProgressBar}>
-      <li>Dane dziecka</li>
-      <li>Wnioskodawcy</li>
-      <li>Zespół</li>
-      <li>Podsumowanie</li>
-    </ul>
-  );
-};
+class ProgressBar extends Component {
```

Comment:
do we need a class component for this? it's only UI, you don't have any states here